corehq/apps/style/static/style/less/_hq/navs.less

```diff
@@ -1,3 +1,7 @@
+#hq-sidebar {
+  min-height: 1000px;
```

Comment:
i don't have too much context here but don't we typically try to avoid defining widths and heights of containers in pixels?